ui-tests/src/test/java/org/openmrs/reference/MergeVisitsTest.java

```diff
@@ -36,13 +35,14 @@ public void setUp() {
     }
 
     @Test
-    @Ignore
     @Category(BuildTests.class)
     public void mergeVisitsTest() {
         FindPatientPage findPatientPage = homePage.goToFindPatientRecord();
         findPatientPage.enterPatient(patient.identifier);
+        findPatientPage.waitForPageToLoad();
         ClinicianFacingPatientDashboardPage clinicianFacingPatientDashboardPage = findPatientPage.clickOnFirstPatient();
         PatientVisitsDashboardPage patientVisitsDashboardPage = clinicianFacingPatientDashboardPage.goToRecentVisits();
+        patientVisitsDashboardPage.waitForPage();
```

Comment:
`Waiting the page` ensures that the `classLoader` avails the  [element](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/b58e4d732d5fc2eca1cd1aafc14a11839ba43863/ui-tests/src/test/java/org/openmrs/reference/page/PatientVisitsDashboardPage.java#L27)  to appear on the page for the next action to be effective. This particular element takes longer to be loaded on the page than others probably because it is a parent node that holds a container with dropdown elements.  Alternatively calling `waiter instance` in the [method](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/510da39098cf0dabf339f51bd825a538277b118f/ui-tests/src/test/java/org/openmrs/reference/page/PatientVisitsDashboardPage.java#L80) that invokes the element would be opted for but the suggest one in is more `efficient and effective` thus overriding the alternative